packages/react-scripts/package.json

```diff
@@ -42,6 +42,8 @@
     "extract-text-webpack-plugin": "2.1.2",
     "file-loader": "0.11.2",
     "fs-extra": "3.0.1",
+    "html-webpack-plugin": "2.28.0",
+    "jest": "20.0.3",
```

Comment:
`"jest": "20.0.4"` is already in packages.json. Should remove this line `"jest": "20.0.3"